modules/021-cni-cilium/images/operator/werf.inc.yaml

```diff
@@ -1,19 +1,45 @@
 {{- $ciliumVersion := "1.14.5" }}
+# Source repo  settings. Delete after clone repo to fox
+{{- $_ := set . "SOURCE_REPO" "https://github.com" }}
```

Comment:
this default isn't needed. SOURCE_REPO is set up from CI

Reply:
It was temporary